docs/cpp/void-cpp.md

```diff
@@ -19,11 +19,17 @@ You can't declare a variable of type **`void`**.
 ## Example
 
 ```cpp
+void  print_num(int num)
+{
+   std::cout << num << std::endl;
+}
+
```

Comment:
We want our sample to include everything needed to compile the code, so in this case, it would need something like the following at the beginning: 

```cpp
// void.cpp
#include <iostream>
using namespace std;
```

Unfortunately, adding this much machinery has the downside of distracting from the simplicity of the sample. If we can avoid dragging in the library, we should. Something like the following captures this idea:

```cpp
// void.cpp

void return_nothing()
{
    // A void function doesn't return a value.
}
```